engines/wintermute/platform_osystem.cpp

```diff
@@ -144,25 +144,34 @@ void BasePlatform::handleEvent(Common::Event *event) {
 // Win32 API bindings
 //////////////////////////////////////////////////////////////////////////
 bool BasePlatform::getCursorPos(Point32 *lpPoint) {
+#ifndef ENABLE_WME3D
```

Comment:
Should there perhaps be an else for these?

Reply:
Not sure about this, at the moment it does work (meaning the mouse is moving correctly so far and selection works) but I haven't checked it in detail